packages/frontend/src/components/UserSettings/AccessTokensPanel/TokensTable.tsx

```diff
@@ -0,0 +1,118 @@
+import { Button, Classes, Dialog } from '@blueprintjs/core';
+import { ApiPersonalAccessTokenResponse, formatDate } from '@lightdash/common';
+import { Button as MantineButton, Paper, Table } from '@mantine/core';
+import { Dispatch, FC, SetStateAction, useEffect, useState } from 'react';
+import { useTableStyles } from '../../../hooks/styles/useTableStyles';
+import {
+    useAccessToken,
+    useDeleteAccessToken,
+} from '../../../hooks/useAccessToken';
+
+const TokenItem: FC<{
+    token: ApiPersonalAccessTokenResponse;
+    setTokenToDelete: Dispatch<
+        SetStateAction<ApiPersonalAccessTokenResponse | undefined>
+    >;
+}> = ({ token, setTokenToDelete }) => {
+    const { description, expiresAt } = token;
+
+    return (
+        <>
+            <tr>
+                <td>{description}</td>
+                <td>
+                    {expiresAt ? formatDate(expiresAt) : 'No expiration date'}
+                </td>
+                <td>
+                    <MantineButton
+                        variant="outline"
+                        size="xs"
+                        color="red"
+                        onClick={() => setTokenToDelete(token)}
+                    >
+                        Delete
+                    </MantineButton>
+                </td>
+            </tr>
+        </>
+    );
+};
+
+export const TokensTable = () => {
+    const { data } = useAccessToken();
+
+    const { classes } = useTableStyles({
+        '& tr td:last-child': {
+            textAlign: 'right',
+        },
+    })();
```

Comment:
ok I was thinking and have something in mind. in createStyles you can have as many classes as you want. you can create one specifically for `textAlign: 'right'` and you can call it `alignRight`

Reply:
and you can use that on whatever td you like in code

Reply:
That's a great alternative, going to try that right now 👍